tests/README.md

```diff
@@ -0,0 +1,32 @@
+# Tests for SST C API
+This directory includes unit tests and integration tests for the SST C API.
+
+## Unit Tests
+
+- `c_crypto_test.c`: Unit tests for functions in `c_crypto.c`.
+
```

Comment:
Please add back the deleted tests.

Reply:
One example added here. bf733179e6e9f893f8d6f6fde8b4e1bca94ed675

Reply:
Another added here. dea9342302a85812876865691e29b268f3241a4c.

Reply:
Another added here. 8377d39d5379b566f1f7da73cd1c85f3739ff1ec.